discovery-infra/test_infra/utils.py

```diff
@@ -555,6 +555,12 @@ def create_ip_address_list(node_count, starting_ip_addr):
     return [str(ipaddress.ip_address(starting_ip_addr) + i) for i in range(node_count)]
 
 
+def create_ip_address_nested_list(node_count, starting_ip_addr):
```

Comment:
can i have some explanation why we need nested list? cause it seems strange

Reply:
Each NIC may have two IP addresses, one for IPv4 and another for IPv6. The IPv4 addresses are static, IPv6 are allocated  by DHCP, but in order for Terraform to insert them the IP addresses of a NIC must be a list (as opposed to a scalar value). @ori-amizur please correct me if I'm wrong.